target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -25,44 +25,53 @@ edgecore,ecw5410)
 nec,wg2600hp)
 	ucidef_set_led_wlan "wlan2g" "WLAN2G" "green:wlan2g" "phy1tpt"
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "green:wlan5g" "phy0tpt"
-	ucidef_set_led_switch "wan" "WAN" "green:active" "switch0" "0x2"
+	ucidef_set_led_netdev "wan" "WAN" "green:active" "wan"
 	;;
 nec,wg2600hp3)
-	ucidef_set_led_switch "wan" "WAN" "green:active" "switch0" "0x2"
+	ucidef_set_led_netdev "wan" "WAN" "green:active" "wan"
 	;;
 netgear,d7800 |\
 netgear,r7500 |\
 netgear,r7500v2 |\
 netgear,r7800)
 	ucidef_set_led_usbport "usb1" "USB 1" "white:usb1" "usb1-port1" "usb2-port1"
 	ucidef_set_led_usbport "usb2" "USB 2" "white:usb2" "usb3-port1" "usb4-port1"
-	ucidef_set_led_switch "wan" "WAN" "white:wan" "switch0" "0x20"
+	ucidef_set_led_netdev "wan" "WAN" "white:wan" "wan"
 	ucidef_set_led_ide "esata" "eSATA" "white:esata"
 	;;
 tplink,ad7200)
 	ucidef_set_led_usbport "usb1" "USB 1" "blue:usb1" "usb1-port1" "usb2-port1"
 	ucidef_set_led_usbport "usb2" "USB 2" "blue:usb3" "usb3-port1" "usb4-port1"
-	ucidef_set_led_switch "wan" "wan" "blue:wan" "switch0" "0x02"
-	ucidef_set_led_switch "lan" "lan" "blue:lan" "switch0" "0x3c"
+	ucidef_set_led_netdev "wan" "wan" "blue:wan" "wan"
+	ucidef_set_led_netdev "lan1" "lan1" "blue:lan" "lan1"
+	ucidef_set_led_netdev "lan2" "lan2" "blue:lan" "lan2"
+	ucidef_set_led_netdev "lan3" "lan3" "blue:lan" "lan3"
+	ucidef_set_led_netdev "lan4" "lan4" "blue:lan" "lan4"
```

Comment:
netdev trigger does not support multiple interfaces, so this won't work.
cc @sch-m

Reply:
Yes, this won't work. As you add all the lan<x> ports to interface "lan", this would be a bridge and you should configure the led like `ucidef_set_led_netdev "lan" "lan" "blue:lan" "br-lan"`

Reply:
> @ddimension can you post a bootlog from a normal swconfig image?

Here's the bootlog from another device:
[nbg6817_bootlog_std.txt](https://github.com/openwrt/openwrt/files/6154715/nbg6817_bootlog_std.txt)

And a bit network debugging:
[log-nbg6817-2.txt](https://github.com/openwrt/openwrt/files/6154718/log-nbg6817-2.txt)

It seems OpenWRT can send packages. But responses are filtered.

If you need more logs or debugging, just tell me what you need.
BTW, I made the initial support for the device. If you need the original source from Zyxel, I should have it somewhere hopefully.

Kind regards